.github/workflows/solidity.yml

```diff
@@ -253,16 +253,16 @@ jobs:
           IS_RELEASE_CORE: ${{ needs.tag-check.outputs.is-release-core }}
           RELEASE_VERSION_CORE: ${{ needs.tag-check.outputs.release-version-core }}
         run: |
-          VERSION=""
-          if [[ "${IS_RELEASE_CORE}" == 'true' ]]; then
-            VERSION="${RELEASE_VERSION_CORE}"
+          version=""
+          if [[ $IS_RELEASE_CORE == 'true' ]]; then
+            version="${RELEASE_VERSION_CORE}"
           else
             echo "::error::No release version found."
             exit 1
           fi
-          PACKAGE_JSON_VERSION="$(jq -r '.version' package.json)"
-          if [[ "${PACKAGE_JSON_VERSION}" != "${VERSION}" ]]; then
-            echo "::error::Version mismatch: package.json version (${PACKAGE_JSON_VERSION}) does not match version computed from tag ${VERSION}"
+          package_json_version="$(jq -r '.version' package.json)"
+          if [[ "$PACKAGE_JSON_VERSION" != "${version}" ]]; then
+            echo "::error version mismatch: package.json version ($package_json_version) does not match version computed from tag ${version}"
```

Comment:
This seems like merge issues, I'd keep the version from default branch

https://github.com/smartcontractkit/chainlink-evm/blob/44ec1c3b5fc9ca269fc3620ecf429d4da42dafc9/.github/workflows/solidity.yml#L256-L265